aten/src/ATen/core/boxing/OperatorKernel.h

```diff
@@ -22,6 +22,7 @@ namespace c10 {
  */
 struct TORCH_API OperatorKernel : public c10::intrusive_ptr_target {
   virtual ~OperatorKernel() = default;
+  virtual bool isPythonOpRegistrationTrampoline() const { return false; }
```

Comment:
is this getting properly overridden in `PythonOpRegistrationTrampoline`? Maybe I'm just missing it.

Reply:
looks like it's not lol